### PR TITLE
fix(templates/show-people): ensure ORCID icon can render multiple times

### PR DESCRIPTION
**Pull request by sidvishnoi**
Fixes https://github.com/w3c/respec/issues/3129

Regression from https://github.com/w3c/respec/pull/3063. Not sure why hyperHTML only rendered last instance.
Filed https://github.com/WebReflection/hyperHTML/issues/401.

### CODE REVIEW

File: src/core/templates/show-people.js [hunk 1]
```diff
@@ -114,7 +93,23 @@ export default function showPeople(persons = []) {
     }
     if (p.orcid) {
       contents.push(
-        html`<a class="p-name orcid" href="${p.orcid}">${orcidIcon} </a>`
+        html`<a class="p-name orcid" href="${p.orcid}"
```
Comment (saschanaz) [inline src/core/templates/show-people.js:96]: I think we want to do `orcidIcon.cloneNode(true)` instead, as the previous code was moving `orcidIcon` around the parents.

Comment (sidvishnoi) [reply src/core/templates/show-people.js:96]: Oh my! What would I do without you!

**Review by saschanaz — APPROVED** [review]
LGTM but do we need a test?

**Review by saschanaz — APPROVED** [review]
👍

**Pull request merged**